central/role/resources/list.go

```diff
@@ -24,56 +24,92 @@ import (
-	AuthPlugin                       = newResourceMetadata("AuthPlugin", permissions.GlobalScope)
-	AuthProvider                     = newResourceMetadata("AuthProvider", permissions.GlobalScope)
-	BackupPlugins                    = newResourceMetadata("BackupPlugins", permissions.GlobalScope)
-	CVE                              = newResourceMetadata("CVE", permissions.NamespaceScope)
-	Cluster                          = newResourceMetadata("Cluster", permissions.ClusterScope)
-	Compliance                       = newResourceMetadata("Compliance", permissions.ClusterScope)
-	ComplianceRunSchedule            = newResourceMetadata("ComplianceRunSchedule", permissions.GlobalScope)
-	ComplianceRuns                   = newResourceMetadata("ComplianceRuns", permissions.ClusterScope)
-	Config                           = newResourceMetadata("Config", permissions.GlobalScope)
-	DebugLogs                        = newResourceMetadata("DebugLogs", permissions.GlobalScope)
-	Deployment                       = newResourceMetadata("Deployment", permissions.NamespaceScope)
-	Detection                        = newResourceMetadata("Detection", permissions.GlobalScope)
-	Group                            = newResourceMetadata("Group", permissions.GlobalScope)
-	Image                            = newResourceMetadata("Image", permissions.NamespaceScope)
-	ImageComponent                   = newResourceMetadata("ImageComponent", permissions.NamespaceScope)
-	ImageIntegration                 = newResourceMetadata("ImageIntegration", permissions.GlobalScope)
+	// Access is the new resource grouping all access related resources.
+	Access = newResourceMetadata("Access", permissions.GlobalScope)
+	// Administration is the new resource grouping all administration-like resources.
+	Administration = newResourceMetadata("Administration", permissions.GlobalScope)
+	Alert          = newResourceMetadata("Alert", permissions.NamespaceScope)
+	CVE            = newResourceMetadata("CVE", permissions.NamespaceScope)
+	Cluster        = newResourceMetadata("Cluster", permissions.ClusterScope)
+	Compliance     = newResourceMetadata("Compliance", permissions.GlobalScope)
+	Deployment     = newResourceMetadata("Deployment", permissions.NamespaceScope)
+	// DeploymentExtensions is the new resource grouping all deployment extending resources.
+	DeploymentExtensions = newResourceMetadata("DeploymentExtensions", permissions.NamespaceScope)
+	Detection            = newResourceMetadata("Detection", permissions.GlobalScope)
+	Image                = newResourceMetadata("Image", permissions.NamespaceScope)
+	ImageComponent       = newResourceMetadata("ImageComponent", permissions.NamespaceScope)
+	// Integration is the new  resource grouping all integration resources.
+	Integration                      = newResourceMetadata("Integration", permissions.GlobalScope)
 	Indicator                        = newResourceMetadata("Indicator", permissions.NamespaceScope)
```

Comment:
Would it make sense to group this one (processIndicator) with DeploymentExtensions too ?

Reply:
+1 ^